pkg/reconciler/buildrun/resources/sources.go

```diff
@@ -55,11 +55,13 @@ func AmendTaskSpecWithSources(
 		}
 	}
 
-	// create the step for spec.sources, this will eventually change into different steps depending on the type of the source
 	if build.Spec.Sources != nil {
+		// inspecting .spec.sources looking for http typed sources to generate the TaskSpec items in
+		// order to handle external artifacts
```

Comment:
nit: for consistency, say remote artifacts instead of external artifacts

Reply:
Sure! It makes sense.